mart/attack/adversary.py

```diff
@@ -202,10 +234,11 @@ def forward(
     @torch.inference_mode(False)
     def examine(
         self,
-        input: Union[torch.Tensor, tuple],
-        target: Union[torch.Tensor, Dict[str, Any], tuple],
-        model: torch.nn.Module,
-        **kwargs
+        adversary: Optional[torch.nn.Module] = None,
+        input: Optional[Union[torch.Tensor, tuple]] = None,
+        target: Optional[Union[torch.Tensor, Dict[str, Any], tuple]] = None,
+        model: Optional[torch.nn.Module] = None,
```

Comment:
Instead of `= None` make these required keyword arguments.

Reply:
Fixed.